tests/lib.rs

```diff
@@ -732,3 +732,40 @@ fn retain() {
     vs.sort();
     assert_eq!(v, &*vs);
 }
+
+#[test]
+fn get_one_short() {
+    let x = ('x', 42);
+
+    let (r, mut w) = evmap::new();
+
+    w.insert(x.0, x);
+
+    assert_match!(r.get_one(&x.0), None);
+
+    w.refresh();
+
+    assert_match!(r.get_one(&x.0).unwrap().as_ref(), ('x', 42));
```

Comment:
Is this the right way to read the inner value?

Reply:
I think you could probably use
```rust
assert_match!(r.get_one(&x.0).as_deref(), Some(('x', 42)));
```
instead, but either is fine!

Reply:
Great idea, I like that much better! Changes in ac34710870690a8d5ea493ccd9feb6749d324626. I didn't use this syntax in the other test so I could still use `contains` on the range. I could probably use `map(Some).any?` or something but I didn't know if that was worth it.